servicepulse/usage.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Usage
+summary: Use ServicePulse to view and measure the usage of an NServiceBus system.
+component: ServicePulse
+reviewed: 2024-05-08
+related:
+  - nservicebus/usage
+---
+
+This section allows for viewing the endpoint usage summary of a system using NServiceBus.
+A usage report can be generated here at license renewal time.
+
+> [!NOTE]
+> The usage data collection functionality requires ServicePulse version 1.39 or later, and ServiceControl version 5.3 or later.
+
+> [!NOTE]
+> If no data is available, go to the [Usage Setup](usage-config.md) screen to diagnose connection problems.
+
+## Generating a usage report
+
+Clicking `Generate Report` generates a usage report file with the detected [endpoints](#viewing-usage-summary-detected-endpoints) and [queues](#viewing-usage-summary-detected-broker-queues). The report includes the [endpoint type](#setting-an-endpoint-type) selections made on screen, and any specified [words to mask](usage-config.md#report-masks) with be obfuscated.
+
+The report **will not** be automatically uploaded or sent to Particular - it is a JSON file that needs to be provided to Particular upon request.
```

Comment:
```suggestion
The report **will not** be automatically uploaded or sent to Particular - it is a JSON file that can be provided to Particular upon request.
```

Reply:
I believe our eula says that the customer must provide a report when asked - it's not an option if the customer is able to generate one.

Reply:
In that case, I'll fall back to a personal preference:

```suggestion
The report **will not** be automatically uploaded or sent to Particular - it is a JSON file that must be provided to Particular upon request.
```